test/CompanyService.Business.UnitTests/Commands/Company/CreateCompanyCommandTests.cs

```diff
@@ -1,344 +1,557 @@
+using LT.DigitalOffice.CompanyService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.CompanyService.Models.Db;
+using LT.DigitalOffice.CompanyService.Models.Dto.Models;
+using LT.DigitalOffice.CompanyService.Models.Dto.Requests;
+using LT.DigitalOffice.CompanyService.Models.Dto.Requests.Company.Filters;
+using LT.DigitalOffice.CompanyService.Validation.Company.Interfaces;
+using LT.DigitalOffice.Kernel.Broker;
+using LT.DigitalOffice.Kernel.Enums;
+using LT.DigitalOffice.Kernel.Exceptions.Models;
+using LT.DigitalOffice.Kernel.Responses;
+using LT.DigitalOffice.Models.Broker.Requests.Message;
+using LT.DigitalOffice.Models.Broker.Requests.User;
+using LT.DigitalOffice.UnitTestKernel;
+using MassTransit;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Moq.AutoMock;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.CompanyService.Business.UnitTests.Commands.Company
+{
+    public class CreateCompanyCommandTests
+    {
+        private AutoMocker _autoMock;
+        private ICreateCompanyCommand _command;
+
+        private CreateCompanyRequest _request;
+        private DbCompany _company;
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            _autoMock = new();
+
+            _command = _autoMock.CreateInstance<CreateCompanyCommand>();
+
+            _request = new()
+            {
+                PortalName = "PortalName",
+                CompanyName = "Name",
+                SiteUrl = "siteurl",
+                IsDepartmentModuleEnabled = true,
+                AdminInfo = new()
+                {
+                    FirstName = "Name",
+                    MiddleName = null,
+                    LastName = "LastName",
+                    Email = "email@email.com",
+                    Login = "MyLogin",
+                    Password = "MyPassword"
+                },
+                SmtpInfo = new()
+                {
+                    Host = "host",
+                    Port = 123,
+                    Email = "email@email.ru",
+                    EnableSsl = true,
+                    Password = "password"
+                }
+            };
+
+            _company = new()
+            {
+                Id = Guid.NewGuid(),
+                CreatedAt = DateTime.UtcNow,
+                IsActive = true,
+                IsDepartmentModuleEnabled = _request.IsDepartmentModuleEnabled,
+                PortalName = _request.PortalName,
+                CompanyName = _request.CompanyName,
+                Description = null,
+                LogoId = null,
+                Tagline = null,
+                SiteUrl = _request.SiteUrl
+            };
+        }
+
+        [SetUp]
+        public void SetUp()
+        {
+            _autoMock.GetMock<IDbCompanyMapper>().Reset();
+            _autoMock.GetMock<ILogger<ICreateCompanyCommand>>().Reset();
```

Comment:
i cannot find where you use ILogger<ICreateCompanyCommand> mock. why is need here to do reset?

Reply:
done